source/developer/dev-setup.rst

```diff
@@ -1,20 +1,3 @@
 .. _dev-setup:
 
-Developer Machine Setup
-=======================
-
-Setup your developer machine for building and testing the Mattermost server and web client. You can choose to not install the Docker container but you must use it for developing and contributing to Mattermost. The container has an Inbucket email server and both MySQL and PostgreSQL databases, all of which are required for running unit tests.
-
-If you don't plan on contributing code to the Mattermost open source project, then you don't need to install Docker. If you're not installing Docker, you can safely skip the Docker installation parts of the setup instructions. However, the ``make run`` and ``make test`` commands won't work.
```

Comment:
We don't note that Docker is optional at https://developers.mattermost.com/contribute/server/developer-setup/.

Should we?

Reply:
No, while it is technically optional it's not really and we should encourage people to use it.